station-flattening-v0/src/main/java/net/modificationstation/stationapi/mixin/flattening/client/MixinWorldRenderer.java

```diff
@@ -35,6 +35,11 @@ private int changeMaxBlockHeight(int value) {
 	private int changeSectionCount(int value) {
 		return level == null ? value : level.countVerticalSections();
 	}
+	
+	@ModifyConstant(method = "playLevelEvent", constant = @Constant(intValue = 0xFF))
+	private int changeBlockLimit(int value) {
```

Comment:
This will break blocks with meta, no?

BaseClientInteractionManager line 36
```java
level.playLevelEvent(2001, i, j, k, blockBase.id + level.getTileMeta(i, j, k) * 256);
```

Reply:
Hm, yes, I will try to fix that

Reply:
Technically meta uses only 16 values, I can pack it into smaller range. There will be 2^24 possible values for blocks

Reply:
Tested, now it works fine and meta is correct